src/modules/RootPart.rb

```diff
@@ -1711,7 +1712,20 @@ def MountPartitions(root_device_current)
 
               Builtins.y2milestone("fstab %1", fstab)
 
-              if !(
+              if reiserfs_in_fstab?(fstab)
+                reiserfs_entries = fstab.find_all {|e| e["vfstype"].include? "reiser" }
```

Comment:
I would expect rubocop to complain about `{|e| ...}` without spaces, but it looks like yast2-update does not uses rubocop :)